<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget{
border-image: url(:/image/C:/Users/lenovo/Desktop/icons/background.jpg);
}
#MainWindow{
background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 106, 145, 214), stop:1 rgba(255, 255, 255, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget{
border-image: url(:/image/icons/background.jpg);
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox{
color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="title">
       <string>BioChip</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>450</y>
         <width>731</width>
         <height>42</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Time:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Exit(&amp;E)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>410</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options(&amp;O)</string>
    </property>
    <addaction name="actionSet_Row_and_Column"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionLast_Step"/>
    <addaction name="actionNext_Step"/>
    <addaction name="actionReset"/>
    <addaction name="actionPlay"/>
    <addaction name="actionOpen_File"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;B)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp_H"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSetTable"/>
   <addaction name="actionStart_2"/>
   <addaction name="actionLast_Frame"/>
   <addaction name="actionNext_Frame"/>
   <addaction name="actionReset_2"/>
   <addaction name="actionCleanable"/>
   <addaction name="actionMusic_on"/>
   <addaction name="actionOpen_File_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start(&amp;S)</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop(&amp;E)</string>
   </property>
  </action>
  <action name="actionLast_Step">
   <property name="text">
    <string>Last Step(L)</string>
   </property>
  </action>
  <action name="actionNext_Step">
   <property name="text">
    <string>Next Step(N)</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset(R)</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play Video(P)</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Load File(L)</string>
   </property>
  </action>
  <action name="actionStart_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/icons/begin.png</normaloff>:/image/icons/begin.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start the process</string>
   </property>
  </action>
  <action name="actionReset_2">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/icons/reset_1.jpg</normaloff>:/image/icons/reset_1.jpg</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset the Process</string>
   </property>
  </action>
  <action name="actionCleanable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/icons/broom_on.png</normaloff>:/image/icons/broom_on.png</iconset>
   </property>
   <property name="text">
    <string>Cleanable</string>
   </property>
   <property name="toolTip">
    <string>The DMFB can be cleaned.</string>
   </property>
  </action>
  <action name="actionNext_Frame">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/icons/next.jpg</normaloff>:/image/icons/next.jpg</iconset>
   </property>
   <property name="text">
    <string>Next Frame</string>
   </property>
   <property name="toolTip">
    <string>Show Next Frame</string>
   </property>
  </action>
  <action name="actionLast_Frame">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/icons/back_1.jpg</normaloff>:/image/icons/back_1.jpg</iconset>
   </property>
   <property name="text">
    <string>Last_Frame</string>
   </property>
   <property name="toolTip">
    <string>Show the last frame.</string>
   </property>
  </action>
  <action name="actionMusic_on">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/icons/Music-icon.png</normaloff>:/image/icons/Music-icon.png</iconset>
   </property>
   <property name="text">
    <string>Music_on</string>
   </property>
   <property name="toolTip">
    <string>Turn on the music.</string>
   </property>
  </action>
  <action name="actionOpen_File_2">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/icons/openfile.jpg</normaloff>:/image/icons/openfile.jpg</iconset>
   </property>
   <property name="text">
    <string>Load_File</string>
   </property>
   <property name="toolTip">
    <string>Open and read the input file.</string>
   </property>
  </action>
  <action name="actionSetTable">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/icons/settable.png</normaloff>:/image/icons/settable.png</iconset>
   </property>
   <property name="text">
    <string>SetTable</string>
   </property>
   <property name="toolTip">
    <string>Set the row and column of table</string>
   </property>
  </action>
  <action name="actionSet_Row_and_Column">
   <property name="text">
    <string>Set Row and Column(&amp;M)</string>
   </property>
  </action>
  <action name="actionHelp_H">
   <property name="text">
    <string>Help(&amp;H)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
